ios-base/DroidKaigi 2020/Session/RealmEntity/SessionEntity.swift

```diff
@@ -0,0 +1,75 @@
+import ios_combined
+import RealmSwift
+
+final class SessionEntity: Object {
+    @objc dynamic var id: String = ""
+    @objc dynamic var title: String = ""
+    @objc dynamic var enTitle: String = ""
+    @objc dynamic var desc: String = ""
+    @objc dynamic var stime: Int = 0
+    @objc dynamic var etime: Int = 0
```

Comment:
I  am using same property name as [`SessionEntity` in the Android app](https://github.com/DroidKaigi/conference-app-2020/blob/master/data/db/src/main/java/io/github/droidkaigi/confsched2020/data/db/internal/entity/SessionEntityImpl.kt) because I think that it is better that the entity stored locally and the entity stored on the API side have the same name and type. So, I used `stime` not `startTime`. How do you think of it?

Reply:
I see. I understand 👍 
I think it's ok to keep this code 😄